### ISSUE TITLE
[BUG] Cannot show tweets from individual accounts

### ISSUE THREAD

**Issue by oguzkarayemis** [jonjomckay/fritter#319]
> Please make sure you've tried the [latest version](https://github.com/jonjomckay/fritter/releases) of Fritter first, as the F-Droid version is sometimes out of date!

**Describe the bug**
A clear description of what the bug is.
When I open a user's individual twits, I cannot see anything in "Tweets" and "Tweets & Replies" tab. Media tab just working. I get this error code: 
`FormatException: Uexpected end of input (at character 1)
^`
**To Reproduce**
1. Go to an feeds
2. Click on an individual account
3. See error

**Screenshots**
![Screenshot_2022-01-21-15-29-27-901_com jonjomckay fritter](https://user-images.githubusercontent.com/80787950/150527844-4fe30467-5297-48df-8612-5c695655e166.jpg)

**Device**
Your device model, OS version, device locale/language and Fritter version.
Redmi Note 10S, MIUI Global 12.5.5, I've v2.16.2+20220120 from Play Store.

**Comment by nnuudev** [comment]
It seems that the old `api.twitter.com/2/timeline/profile/{user_id}.json` endpoint doesn't work anymore, it always returns a 429 error. The endpoint for media posts only still works, though.

In case this may be of help, it seems that this deprecated endpoint can somewhat be replaced by the search endpoint (`api.twitter.com/2/search/adaptive.json`), if we use it to search for the tweets made by some specific user. I think this is how the _Feed_ tab works on Fritter, except that Fritter searches for all the users we saved and here we'd search for one user only.

- Adding `-filter:replies` to the query returns only regular tweets, whereas if this is omitted, the endpoint returns tweets and replies, so this could work as a replacement for both the _Tweets_ and _Tweets & Replies_ tabs.
- Also, when checking some user profile, you'll probably want to see the tweets from other users that have been retweeted by the user you're checking. This can be achieved by adding `include:nativeretweets` to the query.

In short, a query to see the tweets from @EFF, for example, including retweets and omitting replies, would be something like
`...&q=from:eff+-filter:replies+include:nativeretweets&...`

**Comment by jonjomckay** [comment]
It looks like this is happening because Twitter have blocked the key I'm using from accessing that endpoint, but there's a new one floating around the internet. Nitter is using it already, and it looks stable so far, so I think I'm going to migrate Fritter to use that key instead.

I'll ship a fix for this ASAP! Sorry to all affected by this. Twitter doesn't seem to like Fritter very much :)

**Comment by oguzkarayemis** [comment]
Meh, what a sham of them! Fritter is better than their app :)). Thank you so much, We are looking forward to the new update.

**Comment by jonjomckay** [comment]
To anyone who wants to help ensure this is fixed:

I've pushed changes that should fix this issue, and there's an APK currently building here: https://github.com/jonjomckay/fritter/actions/runs/1736368503

Please download it, and report back here whether the issue is fixed for you - thanks!

**Comment by nnuudev** [comment]
Can confirm that this apk works for me.

**Comment by oguzkarayemis** [comment]
I worked for me, too. Thanks.

**Comment by Flash1232** [comment]
Same

**Comment by Flash1232** [comment]
Same. Working as before. Thanks a buch!

**Comment by jonjomckay** [comment]
Thanks all for testing. Shipping v2.16.3 now!

**Comment by durka** [comment]
Sorry for the bother, did you ship it to F-Droid? I know F-Droid can be slow about building updates but I have not been able to update yet to fix this.

**Comment by Aga-C** [comment]
@durka F-Droid irregularly releases new versions of the apps. Fritter is currently in the build queue, we just need to wait for the release, maybe a day or two.
